.gitignore

```diff
@@ -7,5 +7,7 @@ runtime/wasm/target/
 .idea
 .vscode
 polkadot.*
+!polkadot.service
```

Comment:
Why should systemd service units be in `.gitignore`?

Reply:
This is actually a negation to counteract the `polkadot.*` line directly above - I didn't want to simply remove that line without understanding the potential consequences.